src/Object/Category.php

```diff
@@ -36,39 +36,11 @@
 
 namespace Nosto\Object;
 
-use Nosto\AbstractObject;
-use Nosto\Types\MarkupableInterface;
-
 /**
  * Category object for tagging
  */
-class Category extends AbstractObject implements MarkupableInterface
+class Category extends MarkupableString
```

Comment:
Instead of overriding the getMarkupKey() method, is it better to have a constructor that set the markupkey? 
 `   public function __construct($category)
    {
        parent::__construct($category, 'nosto_category');
    }
`

Reply:
Much better pattern, thanks.